dockers/docker-syncd-cavm/start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export XP_ROOT=/usr/bin/
+
+service rsyslog start
+syncd -p /etc/ssw/AS7512/profile.ini -N
```

Comment:
Can you enable the counter collecting thread on your platform? It is important for sonic to get those counters from asic.